test/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/UKAddressLookupServiceImplSpec.scala

```diff
@@ -119,6 +119,120 @@ class UKAddressLookupServiceImplSpec extends WordSpec with Matchers with MockFac
           ).isLeft shouldBe true
         }
 
+        "address line1 is more than 35 characters" in {
```

Comment:
the service isn't returning an error in this case - we should have a different section of the spec which is something like:
```
  "filter out invalid addresses" when { 
    "address line1 is more than 35 characters" in { ...}
  }
```